structure/0062-quilt-community-collab.md

```diff
@@ -0,0 +1,135 @@
+# Summary
+
+This document describes the Quilt Community Collab program, the roles of each member and the associated processes.
+
+# Motivation
+
+The Minecraft community is a complex space, where it can be required for moderators to share intel between communities. The main goal of the Quilt Community Collab program is to facilitate exchange between a selected group of communities and improve user safety.
+
+The server also serves as a coordination server for events and other projects happening in the Minecraft sphere.
+
+# Explanations
+
+The Quilt Community Collab is a private space for Minecraft-centered communities to exchange information and coordinate. The main focus of the exchanges is set on moderation, but it can also be used for events and anything else that may be fitting.
+
+## What Community Collab isn’t
+
+- It isn’t an official affiliation with the Quilt project itself or any other community part of the program
+- It isn’t a place to blindly synchronise punishments between communities
+    - Each community must decide on appropriate moderation action based on facts shown to them and their own rules and policies. Infringing this rule may get your community removed from the Community Collab program.
+
+## Roles inside the server
+
+### Keyholder
+
+The keyholder for this space is the same as for the Quilt Community Team. This role is described in the [RFC 0007 # Keyholders](https://github.com/QuiltMC/rfcs/blob/master/structure/0007-community-team.md#keyholders).
+
+### Managers
```

Comment:
Note to self: managers should be given moderation power if it is really required.